srcpkgs/swaylock/patches/fix-shm_unlink-undefined-reference.patch

```diff
@@ -0,0 +1,29 @@
+From d9e71c87c7af8687750901d71409140f1bd181bd Mon Sep 17 00:00:00 2001
+From: nerdopolis <bluescreen_avenger@verizon.net>
+Date: Sun, 27 Nov 2022 18:46:05 -0500
+Subject: [PATCH] Fix build on Debian Stable
```

Comment:
Taken from https://github.com/swaywm/swaylock/commit/d9e71c87c7af8687750901d71409140f1bd181bd. Should I change the commit message? Should I also note that this patch should be removed on the next release, and if so, how?

Reply:
fine as is